boards/x86/intel_rpl/intel_rpl_s_crb_defconfig

```diff
@@ -15,3 +15,4 @@ CONFIG_BUILD_OUTPUT_EFI=y
 CONFIG_BUILD_NO_GAP_FILL=y
 CONFIG_HEAP_MEM_POOL_SIZE=64000000
 CONFIG_MAIN_STACK_SIZE=320000
+CONFIG_MAX_THREAD_BYTES=11
```

Comment:
are you sure it is needed here?

Reply:
That's counting total threads, not CPUs (it allows for 88 threads total).  You're saying the per-CPU idle threads (the only ones that scale with core count) are pushing it across the limit?  Please clarify in the patch description.  The default value is "2", so this is adding space for 72 new threads, not the "six more rounded up to a multiple of 8" I would expect. I agree this looks suspicious.